_posts/2024-05-05-Jonathan-O-Darkreader-Fix.md

```diff
@@ -0,0 +1,63 @@
+---
+layout: post
+# If your post title is longer or more complicated
+# than can be represented in the filename, uncomment the following line
+# and specify a custom title
+# title:  "Sample Blog Post"
+
+# Uncomment only one of the below categories
+categories: 
+Bug Fix
```

Comment:
```suggestion
- Bug Fix
```
small metadata typo is causing jekyll to freak out

Reply:
Thank you, should be fixed now.